src/rulesets/Base/fastmath_able.jl

```diff
@@ -33,6 +33,11 @@ let
         @scalar_rule log10(x) inv(x) / log(oftype(x, 10))
         @scalar_rule log1p(x) inv(x + 1)
         @scalar_rule log2(x) inv(x) / log(oftype(x, 2))
+        
+        # rouding related
```

Comment:
```suggestion
        # rouding related, 
        # we use `zero` rather than `Zero()` because cheaper for scalar, and avoids issues with map etc
```